tests/e2e/scenarios/test_reborn_webui_v2_legacy_sse_history.py

```
@@ -0,0 +1,1028 @@
+    headers = {"Authorization": f"Bearer {REBORN_V2_AUTH_TOKEN}"}
+    async with httpx.AsyncClient(headers=headers) as client:
+        thread_id = await _create_thread(client, reborn_v2_server)
+
+    events_path = f"/api/webchat/v2/threads/{thread_id}/events"
+    events_url = f"{reborn_v2_server}{events_path}"
+    params = {"token": REBORN_V2_AUTH_TOKEN}
+    request_headers = {"Accept": "text/event-stream"}
+    timeout = aiohttp.ClientTimeout(total=15, sock_read=15)
+
+    async with aiohttp.ClientSession(timeout=timeout) as session:
+        async with AsyncExitStack() as stack:
```

Ruff (0.15.20)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)